notebooks/worksheet2.ipynb

```diff
@@ -115,15 +115,16 @@
    "source": [
     "# Provide the names of the directories where the netCDF model files are stored\n",
     "DATADIR = 'data_v2/'\n",
+    "DOMAIN = 'EAS-22'\n",
     "\n",
     "# Load and concatenate the HadGEM2-ES model cube data\n",
-    "infile = os.path.join(DATADIR, 'EAS-22/pr_EAS-22_MOHC-HadGEM2-ES_historical_r1i1p1_GERICS-REMO2015_v1_mon_*.nc')\n",
+    "infile = os.path.join(DATADIR, DOMAIN, 'pr_'+DOMAIN+'_MOHC-HadGEM2-ES_historical_r1i1p1_GERICS-REMO2015_v1_mon_*.nc')\n",
```

Comment:
PEP8 we should have whitespaces either side of the `+` next to `DOMAIN`, perhaps it's own PR.

Reply:
yes PEP8 compliance should be it's own PR - harder to enforce this in notebooks though as I don' t think most checkers can run on notebooks?

Reply:
black can run on notebooks https://black.readthedocs.io/en/stable/getting_started.html#installation... if you have time it might be worth setting up a pre-commit CI that uses black every time new commits are made... see iris repo for example